src/Console/Commands/Make/Modularize.php

```diff
@@ -68,7 +68,7 @@ protected function getPath($name)
 			
 			// Normalize all our paths for compatibility's sake
 			$normalize = function($path) {
-				return DIRECTORY_SEPARATOR.trim($path, DIRECTORY_SEPARATOR).DIRECTORY_SEPARATOR;
+				return rtrim($path, '/').'/';
```

Comment:
Is the switch from `trim()` to `rtrim()` necessary? If so, how can we be sure this won't negatively impact absolute linux paths?

Reply:
We don't. If there is no leading separator this will produce different results. It should be `'/'.trim().'/'`.

My PC is down ATM - waiting for new power supply. Please feel free to change this.